cmd/admin-service.go

```diff
@@ -38,21 +38,14 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 COMMANDS:
-   {{range .VisibleCommands}}{{join .Names ", "}}{{ "\t" }}{{.Usage}}
-   {{end}}
+  {{range .VisibleCommands}}{{join .Names ", "}}{{ "\t" }}{{.Usage}}
```

Comment:
Not everything is common but some of them are similar.
E.g, 
- cp-main and cat-main have the same template.
- ls-main and mb-main have the same template.
We can reduce duplicating the string and be consistent.